scripts/git-auth.sh

```
@@ -0,0 +1,155 @@
+      return 0
+      ;;
+  esac
+
+  GIT_AUTH_TOKEN_LABEL=""
+  GIT_AUTH_TOKEN_VALUE=""
+  git_auth_resolve_token "$remote_url" "$default_token_var" || return 0
+  token="$GIT_AUTH_TOKEN_VALUE"
+  token_label="$GIT_AUTH_TOKEN_LABEL"
+  [[ -n "$token" ]] || return 0
+  header="$(git_auth_basic_header "$user" "$token")"
+  GIT_AUTH_ENV=(
```

Shellcheck (0.11.0)

[warning] GIT_AUTH_ENV appears unused. Verify use (or export if used externally).

(SC2034)